charts/README.md

```diff
@@ -33,6 +33,21 @@ $ kubectl get secret -n karmada-system karmada-kubeconfig -o jsonpath={.data.kub
 
 > **Tip**: List all releases using `helm list`
 
+For better scheduling effect, you can choose to install the `karmada-descheduler` component in the karmada controller plane. The following is the specific installation guide.
```

Comment:
Add a document link here woud be better.


For better scheduling effect, you can choose to install the [karmada-descheduler](../docs/descheduler.md) component in the karmada controller plane. The following is the specific installation guide.

Reply:
Link: (../docs/descheduler.md)